views/popups.pug

```diff
@@ -230,3 +235,39 @@
                 label
                   input.hideEditorToolbars(type="checkbox")
                   | Hide Editor Toolbars
+
+#history.modal.fade.gl_keep(tabindex="-1" role="dialog")
+  .modal-dialog.modal-lg
+    .modal-content
+      .modal-header
+        h5.modal-title History
+        button.close(type="button" data-dismiss="modal" aria-hidden="true" aria-label="Close")
+          span(aria-hidden="true")
+            | &times;
+      .modal-body
+        .card
+          .card-body
+            ul.nav.nav-tabs(role="tablist")
+              li.nav-item
+                a.nav-link.active#loadhistory-tab(data-toggle="tab" role="tab" href="#load-history" aria-controls="load-history" aria-selected="true")
```

Comment:
As noted above, can we make these use `class` instead of `id`-based `#` names? It's not a huge problem, but using IDs means worrying about clashing IDs. I mean, clashing classes is a problem as well but at least they're somewhat scoped (and clashing IDs can cause weird browser issues). Anyway, no big deal but thought I'd mention!